app/components/djs/list.hbs

```diff
@@ -8,34 +16,60 @@
       </h1>
     </await.Pending>
     <await.Fulfilled as |result|>
-      <div
-        class="djs flex flex-wrap "
-      >
+
+
+      <div class="grid grid-cols-fruits gap-6">
         {{#each result as |dj|}}
-          <div class="dj m-5 classic:bg-df-pink blm:bg-black trans:bg-df-blue rounded-md">
-            <h2 class="name font-debussy text-xl m-2 overflow-hidden">
-              <LinkTo @route="home.dj" @model={{dj.username}} title={{dj.username}}>
-                {{dj.username}}
-              </LinkTo>
-            </h2>
-            <User::Badges @role={{dj.role}} />
+
+          {{! User }}
+          <div class="flex
+                  flex-col
+                  gap-2
+                  mb-2
+                  p-2
+                  
+                  classic:bg-df-blue-dark
+                  blm:bg-black
+                  trans:bg-df-blue
+                  
+                  rounded-xl
+                  overflow-hidden"
+                  >
+            
+            {{! User profile picture }}
             {{#if (eq dj.avatarUrl null)}}
               <img
-                width="150"
-                height="150"
-                class="rounded-b-md align-bottom"
+                style="aspect-ratio:1/1;"
+                class="width-full rounded-md"
```

Comment:
looks like tailwind has `aspect-square` https://tailwindcss.com/docs/aspect-ratio

Reply:
It does and I tried it, but I couldn't get it to work on the actual production site. I had wondered if the version of Tailwinds the site uses isn't up to date or something? Or maybe there's something I missed.

Reply:
Ah ok no worries, not a big deal just thought I'd make a note of it. Maybe our tailwind version is old.